installer/install-golangci-lint-langserver.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+
+"$(dirname "$0")/go_install.sh" github.com/nametake/golangci-lint-langserver@latest
```

Comment:
Prebuilt binaries are avaliable. https://github.com/nametake/golangci-lint-langserver/releases

Reply:
Let's merge this and will fix installer when the Windows binary will be released.